phpBB/styles/prosilver/template/mcp_header.html

```diff
@@ -19,31 +19,31 @@
 <div class="panel bg3">
 	<div class="inner">
 
-	<div style="width: 100%;">
+		<div style="width: 100%;">
 
-	<div id="cp-menu">
-		<div id="navigation" role="navigation">
-			<ul>
-			<!-- BEGIN l_block1 -->
-			<!-- IF l_block1.S_SELECTED -->
-				<!-- BEGIN l_block2 -->
-				<!-- IF l_block1.l_block2.S_SELECTED -->
-				<li id="active-subsection"><a href="{l_block1.l_block2.U_TITLE}"><span>{l_block1.l_block2.L_TITLE}<!-- IF l_block1.l_block2.ADD_ITEM --> ({l_block1.l_block2.ADD_ITEM})<!-- ENDIF --></span></a></li>
-				<!-- ELSE -->
-				<li><a href="{l_block1.l_block2.U_TITLE}"><span>{l_block1.l_block2.L_TITLE}<!-- IF l_block1.l_block2.ADD_ITEM --> ({l_block1.l_block2.ADD_ITEM})<!-- ENDIF --></span></a></li>
-				<!-- ENDIF -->
-				<!-- END l_block2 -->
-			<!-- ENDIF -->
-			<!-- END l_block1 -->
-			</ul>
-		</div>
-	</div>
+			<div id="cp-menu" class="cp-menu">
```

Comment:
hmm not sure i understand, are you meaning merge conflicts for patches? either way the fact that the indentation is all over the place and inconsistent is a constant problem when making changes and needs to be fixed I guess we need a scrub PR to fix all this. Thats sort of a daunting task though with template inclusion